src/app/Dice.elm

```diff
@@ -0,0 +1,19 @@
+module Dice exposing (emptyCup, Cup, cupFromIntList)
+
+
+type alias Die =
+    Int
+
+
+type alias Cup =
```

Comment:
Personally, this type alias would be the only thing I'd keep in this file. I don't have particularly strong views about it, but after messing around with `type alias Dice = List Die` related stuff, I did not see enough merits to add this level of meaning.